### PR TITLE
findex: 0.7.1 -> 0.7.2

### PR DESCRIPTION
**Pull request by aquifolly**
###### Description of changes

https://github.com/mdgaziur/findex/releases/tag/v0.7.2

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/findex/default.nix [hunk 1]
```diff
@@ -1,32 +1,48 @@
...
   ];
 
   meta = with lib; {
     description = "Highly customizable application finder written in Rust and uses Gtk3";
-    homepage = "https://github.com/mdgaziur/findex";
+    homepage = "https://github.com/mdgaziur/Findex";
     license = licenses.gpl3Only;
     platforms = platforms.linux;
-    maintainers = [ maintainers.pinkcreeper100 ];
+    maintainers = with maintainers; [ maintainers.pinkcreeper100 ];
```
Comment (AndersonTorres) [inline pkgs/applications/misc/findex/default.nix]: Do you understand this line?

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/applications/misc/findex/default.nix (hunk 1 above)
Comment (pinkcreeper100) [reply pkgs/applications/misc/findex/default.nix]: yes, I have changed that line in a later revision

**Comment by AndersonTorres** [comment]
Broken on Darwin.

**Comment by pinkcreeper100** [comment]
It's not intended for darwin
https://github.com/pinkcreeper100/nixpkgs/blob/86e58f56f409248383ca2e24a4f101b4b6eac374/pkgs/applications/misc/findex/default.nix#L45

**Comment by pinkcreeper100** [comment]
and the linux builds fail because upstream forgot to update the cargo lockfile

**Comment by AndersonTorres** [comment]
1. Recreate the lockfile and use `cargoLock.lockfile = ./Cargo.lock`.
    Also in the postPatch substitute the lockfile.

2. Add `meta.badPlatforms = [ "aarch64-darwin" "x86_64-darwin" ];`

**Comment by pinkcreeper100** [comment]
I'd preferably just wait until it's fixed upstream

**Comment by pinkcreeper100** [comment]
added `meta.badPlatforms`

**Comment by AndersonTorres** [comment]
OK. I will put this on draft mode until they sort out that new lockfile.

**Comment by pinkcreeper100** [comment]
decided to make a pr to update it myself

**Comment by pinkcreeper100** [comment]
I've kinda given up on this, as upstream development has halted

**Pull request closed without merging**